assets/src/containers/FilesAccessed.js

```diff
@@ -160,9 +179,27 @@ function FilesAccessed (props) {
     }
   }
 
+  const onChangeFileHandler = event => {
+    const value = event.target.value
+    if (event.target.checked == true && !resourceFilter.includes(value)) {
+      setResourceFilter([...resourceFilter, value])
+    } 
+    else if (event.target.checked == false) { 
+      setResourceFilter(resourceFilter.filter(val => {
+        return val != value
+      }))
+    }
+    
+  }
+
   const FileAccessChartBuilder = (fileData) => {
     if (!fileData || Object.keys(fileData).length === 0) {
-      return (<p>No data provided</p>)
+      if (resourceFilter == "") {
```

Comment:
Empty quotes are considered falsy in JavaScript, so this can simply be `if (!resourceFilter)`

Reply:
Didn't work properly

Reply:
Hmm strange. Maybe just use `resourceFilter === ''` instead.

Reply:
I did `if (resourceFilter <= [])`. Is this fine?

Reply:
less than operator should be used for comparing the number and in this case, it is comparing with an empty list. I don't think that makes sense

Reply:
I can't do `resourceFIlter == []` or `resourceFilter === []` since they always evaluate to false. Is there another way?

Reply:
Try `resourceFilter.length == 0` ?

Reply:
Thanks